atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampBoundStore.java

```diff
@@ -155,6 +163,12 @@ private void cas(Client client, Long oldVal, long newVal) {
         }
     }
 
+    private String getCurrentValues(CASResult result) {
```

Comment:
Maybe something like `getCurrentTimestampValues` or something would be a bit better. I didn't really understand what this was extracting at first.